e2e-tests/test_config.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2019 The Ground Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the 'License');
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an 'AS IS' BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LoiType} from './ground-helpers';
+
+export class TestConfig {
+  static WEB_URL = 'http://localhost:5000';
+  static TEST_TIME_OUT = 5 * 60 * 1000; // 5m
```

Comment:
"Time-out" is generally written as one word in js + ts (as in `setTimeout`). In that case, these static constants would be rendered as `*_TIMEOUT`.

Reply:
Done.